src/zmalloc.c

```diff
@@ -686,26 +686,8 @@ int zmalloc_get_allocator_info(size_t *allocated, size_t *active, size_t *reside
      * into account all allocations done by this process (not only zmalloc). */
     je_mallctl("stats.allocated", allocated, &sz, NULL, 0);
 
-    /* Retained memory is memory released by `madvised(..., MADV_DONTNEED)`, which is not part
-     * of RSS or mapped memory, and doesn't have a strong association with physical memory in the OS.
-     * It is still part of the VM-Size, and may be used again in later allocations. */
-    if (retained) {
-        *retained = 0;
-        je_mallctl("stats.retained", retained, &sz, NULL, 0);
-    }
-
-    /* Unlike retained, Muzzy representats memory released with `madvised(..., MADV_FREE)`.
-     * These pages will show as RSS for the process, until the OS decides to re-use them. */
-    if (muzzy) {
-        size_t pmuzzy, page;
-        assert(!je_mallctl("stats.arenas." STRINGIFY(MALLCTL_ARENAS_ALL) ".pmuzzy", &pmuzzy, &sz, NULL, 0));
-        assert(!je_mallctl("arenas.page", &page, &sz, NULL, 0));
-        *muzzy = pmuzzy * page;
-    }
```

Comment:
why did you remove muzzy? we do want to show it in INFO.
p.s. maybe we can keep the code that extracts retained, with the comment, just as a reminder what it is.

Reply:
sorry, i misunderstood what you meant, i've reverted the changes.